manifests/ci-service-account/README.md

```diff
@@ -0,0 +1,11 @@
+# Service account for CI Bot
```

Comment:
Can you add appropriate section to the main readme file, that will describe the use case for this manifest and how/when to use it + any dependencies?

Reply:
I can add some extra paragraphs about why this exists to the root README for sure.  One question though.
Do you want me to remove this readme from _this_ directory and place its contents in the README in the root repo?